server/openapi_server/models/DB_utils.py

```
@@ -0,0 +1,620 @@
+import uuid
```

Ruff (0.11.9)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)